BookPlayer/Library/LibraryViewController.swift

```diff
@@ -294,23 +298,24 @@ class LibraryViewController: BaseListViewController, UIGestureRecognizerDelegate
 }
 
 // MARK: - TableView Delegate
+
 extension LibraryViewController {
-    func tableView(_ tableView: UITableView, editActionsForRowAt indexPath: IndexPath) -> [UITableViewRowAction]? {
+    func tableView(_: UITableView, editActionsForRowAt indexPath: IndexPath) -> [UITableViewRowAction]? {
```

Comment:
interesting 🤔 I didn't know you could just leave '_' w/o specifying the name of the parameter in the function declaration if it isn't being used